schema.sql

```diff
@@ -1,5 +1,42 @@
-/* Database schema to keep the structure of entire database. */
+/* 
+Database schema to keep the structure of entire database.
 
+** hope you are connected with data base **
+if not open psql shell
+Server [localhost]:
+Database [postgres]: --you can write here you database name as well
+Port [5432]:
+Username [postgres]:
+Password for user postgres:
+** output**
+psql (15.2)
+WARNING: Console code page (437) differs from Windows code page (1252)
+         8-bit characters might not work correctly. See psql reference
+         page "Notes for Windows users" for details.
+Type "help" for help.
+*/
+
+\c vet_clinic
+
+/*
+** output **
+You are now connected to database "vet_clinic" as user "postgres".
+ */
+
+-- creating animal table
 CREATE TABLE animals (
-    name varchar(100)
+  id int, --i thought i should add serial but seen int in project
+	name int,
```

Comment:
- [ Required ] I think it would be better if the `name` field in the `animals` table is given the `varchar(255)` data type, the `int` data type is usually used to represent `numeric` data. You can achieve this by refactoring this code as shown below.

```SQL
    CREATE TABLE animals (
      id int,
      name varchar(255),
      date_of_birth date,
      escape_attempts int,
      neutered boolean,
      weight_kg decimal,
    );
```

Reply:
# Thanks for the typo 


i think i made data base with varchar(30) or charector(30) ;

see below for ss

![image](https://user-images.githubusercontent.com/116451144/230284829-bb8be2d1-aca9-410c-820a-011a911c0bba.png)